app/lib/test_executor.rb

```diff
@@ -0,0 +1,66 @@
+class TestExecutor
+  HOST_WORKDIR = '/usr/src/app'
+  WORKDIR = '/tmp/sandbox'
+
+  def self.images
+    @images ||= {}
```

Comment:
You would want this to be `@@images` because it's a static variable. Not sure how `@images` works in a static method.